docs/logs/extending-the-sdk/Program.cs

```diff
@@ -37,9 +39,6 @@ public static void Main()
         // unstructured log
         logger.LogInformation("Hello, World!");
```

Comment:
maybe leave both the unstructure logs lines as-is, but add a comment above saying "unstructured logging is not recommended, but shown here for comparison"..?

Reply:
or have the whole unstructured log statements as commented out, so CA2254 won't complain, but it serves the purpose of showing the difference between structured and unstructured.

Reply:
If we uncomment it, it'd be good to either add a `[SuppressMessage]` to turn that off for that line or `#pragma warning disable` around it. Or, as you mentioned, I could put it back and leave it commented. It did seem odd to show "you can do this!" for things that you really shouldn't be doing so it felt like removing it was the better option for a documentation project. If it was unit tests then the suppression would be more accurate, I think, to retain the test case.

Reply:
> I could put it back and leave it commented.

I'd vote for this approach. 
```
// String interpolation, as in the below line, results in unstructured logging, and is not recommended
logger.LogInformation($"Hello from potato {0.99}.");
```

This way, if someone is doing this, its easy to point them to this doc. Having the incorrect way in a comment block will be useful for them to know what they are doing incorrectly.

Reply:
OK, I put this back in as a comment.